chemicaltoolbox/autodock_vina/prepare_receptor/prepare_receptor.xml

```diff
@@ -0,0 +1,43 @@
+<tool id="prepare_receptor" name="Prepare receptor" version="0.1.0">
+    <requirements>
+        <requirement type="package" version="1.5.6">mgltools</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" />
+    </stdio>
+    <description> - Tool for prepare receptor with Autodock Vina</description>
```

Comment:
`planemo lint` will complain here I guess. description needs to be before requirement.